src/list.c

```diff
@@ -216,6 +216,18 @@ json_t *list_checksum(rcComm_t *conn, rodsPath_t *rods_path,
     results = do_query(conn, query_in, obj_format.labels, error);
     if (error->code != 0) goto error;
 
+    size_t result_size = json_array_size(results);
+    if( result_size > 1 ){
+        for( int i = result_size - 1; i >= 0; i--){
+            json_t *obj = json_array_get(results, i);
+            char *repl_status;
+            json_unpack(json_object_get(obj, JSON_REPLICATE_STATUS_KEY), "s", &repl_status);
+            if (atoi(repl_status) != 1){
+                json_array_remove(results, i);
+            }
+        }
+    }
+
```

Comment:
This is returning the stale replicas and then filtering them out of the results, which as you've demonstrated, does work. It would be better to _not_ return them at all, by refining the query.